### ISSUE TITLE
VS Code autocomplete (intellisense)

### ISSUE THREAD

**Issue by carlosabella** [joeferraro/MavensMate#871]
* **MavensMate Desktop Version**: v0.0.11-beta7 (last I found)

* **Editor**: Code

* **MavensMate Plugin Version**: 0.6.2

* **Platform/Version**: Windows 7 64-bit

* **Proxy Details**: no proxy

* **Salesforce API Version**: 36

* **Salesforce Server**: cs3

Hi, I'm starting with mavensmate and VSCode plugin and by now it works perfectly. I only have a doubt:  I have Auto Completion for apex methods, but I can't get field objects population. 

For example if I declare a new instance of an Account, I can't get field population. Example:
Account a = new Account();
a. 

When I write the "a." I don't get any field. 

Am I doing anything wrong? 

Thank you very much!

**Comment by ChuckJonas** [comment]
Last time I checked Mavensmate didn't support Auto-complete on vscode.  I'm not sure if there are plans to add it or not.

I wrote a separate [auto-complete extension](https://marketplace.visualstudio.com/items?itemName=chuckjonas.apex-autocomplete) for vscode.  

I recently added completion support for `SObjects`, but you must add ORG credentials to the workspace settings (class completion should work without creds)
